civihr_default_theme/scss/base/_variables.scss

```diff
@@ -116,8 +116,8 @@ $nav-tabs-justified-active-link-border-color: $light !default;
 // Media queries breakpoints
 // ------------------------------------------------------------------------------------------------------
 
-$screen-sm-max:                   767px !default;
-$crm-menu-responsive-breakpoint:  1250px;
+$screen-sm-max:     767px !default;
+$crm-screen-lg-max: 1300px;
```

Comment:
I am not sure of the role of this variable, is this meant for only the Menu's breakpoint? 
We already have `$screen-lg-desktop` or `$screen-lg`, is this meant to replace those?

Reply:
no, this is a custom breakpoint which is even bigger than that, as the value of `$screen-lg` (1200px) wasn't high enough. Currently it's used only for the menu, but it can be used in the future for other stuff if we need to